### PR TITLE
fix popup timeout source id is not found on destroy

### LINKED ISSUE #176: Investigate log warning message
The log says "Source ID xxx was not found when attempting to remove it" whenever the popup is destroyed. Strangely this does not happen if the popup is opened with `toggle=true`.

### PR DESCRIPTION
**Pull request by ebeem**
fixes #176

### CODE REVIEW

**Comment by ebeem** [comment]
can't reproduce, but for some reason `this._initialDelayTimeoutId` is = 0 sometimes and this causes the opacity to stay 0
```js
    _showImmediately() {
        if (this._initialDelayTimeoutId === 0)
            return;

        GLib.source_remove(this._initialDelayTimeoutId);
        this._initialDelayTimeoutId = 0;

        Main.osdWindowManager.hideAll();
        this.opacity = 255;
    }
```

File: wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js [hunk 1]
```diff
@@ -96,7 +96,14 @@ class WorkspaceSwitcherPopup extends SwitcherPopup.SwitcherPopup {
         }
 
         if (this._popupTimeout > 0 && !this._toggle) {
-            this._noModsTimeoutId = GLib.timeout_add(GLib.PRIORITY_DEFAULT, this._popupTimeout + 150, this._finish.bind(this));
+            this._noModsTimeoutId = GLib.timeout_add(
+                GLib.PRIORITY_DEFAULT,
+                this._popupTimeout,
```
Comment (mzur) [inline wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js:101]: Was the `+ 150` removed on purpose?

```suggestion
                this._popupTimeout + 150,
```

Comment (ebeem) [reply wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js:101]: Yes, it added an extra delay to wait for popup.
We don't need it now as there is no delay anymore. This was discussed somewhere but I can't find it.

**Review by mzur — APPROVED** [review]
Thanks!

**Pull request merged**